docs/docker.md

```diff
@@ -1,31 +1,28 @@
 # Run in a development Docker container
 
-An alternative way to run the application for development purposes is in a Docker container. This is useful if you don’t
-want to install Node on your host machine. You can also use this method if you want to run the application on an
-[Apple silicon](https://support.apple.com/en-gb/HT211814) computer without emulation, as our ready-made Docker images
-don’t yet support M1.
+An alternative way to run the application for development purposes is in a Docker container. This is useful if you don’t want to install Node on your host machine. 
 
 Start by cloning the repository:
 
     git clone https://github.com/netwerk-digitaal-erfgoed/network-of-terms.git
     cd network-of-terms    
 
-Then start run the development container:
+To run the GraphQL API run the development container using the following command:
 
-    docker compose run --rm node
+    docker compose -f docker-compose-graphql.yml run -p 8080:3123 --rm grapql
 
-And execute the commands that you find in each package’s readme (for example
-[GraphQL](../packages/network-of-terms-graphql/) or [Reconciliation](../packages/network-of-terms-reconciliation)):
+The API will (after a few seconds) be available on localhost:8080. 
+You can change the port number 8080 to your own preference. 
 
-    # In the container:
-    /app # npm install
-    /app # cd packages/network-of-terms-graphql
-    /app/packages/network-of-terms-graphql # npm run dev
+To run the Reconciliaton API run the development container using the following command:
 
-    # or:
+    docker compose -f docker-compose-reconciliation.yml run -p 8080:3123 --rm reconciliation
 
-    /app # cd packages/network-of-terms-reconciliation
-    /app/packages/network-of-terms-reconciliation # npm run dev
+Note that you need to restart the docker compose after every change in the code. 
```

Comment:
That’s too cumbersome for developers, which is why I suggested `npm run dev` (which watches for changes) before.